std_msgs/Header header
ultralytics_ros/Det3D[] detections